pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -297,20 +297,26 @@ func getPhase1Binding(vmi *v1.VirtualMachineInstance, iface *v1.Interface, netwo
 }
 
 func getPhase2Binding(vmi *v1.VirtualMachineInstance, iface *v1.Interface, network *v1.Network, domain *api.Domain, podInterfaceName string) (BindMechanism, error) {
-	populateMacAddress := func(vif *VIF, iface *v1.Interface) error {
+	populateMacAddress := func(iface *v1.Interface) (*net.HardwareAddr, error) {
```

Comment:
Can we refactor the name of the function `populateMacAddress` to something like `retrieveMacAddress`? since after your changes the function no more populating a struct but returns a MAC

Reply:
done

Reply:
@AlonaKaplan Thank you!